action.yml

```diff
@@ -21,8 +21,17 @@ inputs:
   previews:
     description: A comma-separated list of API previews to accept
   result-encoding:
-    description: Either "string" or "json" (default "json")—how the result will be encoded 
+    description: Either "string" or "json" (default "json")—how the result will be encoded
     default: json
+  retries:
+    description: "The number of times to retry a request"
+    default: "0"
+  retry-after:
+    description: "The number of seconds between retries. No effect unless `retries` is set."
+    default: "0"
+  do-not-retry:
```

Comment:
This might be confusing with the "retries" option, my initial read was that this was a boolean to control whether we'd allow retries or not, rather than a set of exempt status codes.

Maybe `retry-status-codes` or something similar?

Reply:
yeah that's fair. I was trying to keep naming consistent with octokit, but that's a good point.